Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1677,12 +1681,29 @@ void WebProcess::nonVisibleProcessEarlyMemoryCleanupTimerFired()
     if (!m_pagesInWindows.isEmpty())
         return;
 
+    destroyDecodedImageData();
+
 #if PLATFORM(COCOA)
     destroyRenderingResources();
     releaseSystemMallocMemory();
 #endif
 }
 
+void WebProcess::destroyDecodedImageData()
+{
+    for (auto& page : m_pageMap.values())
+        page->releaseAllRemoteImageResources();
```

Comment:
It is difficult to keep track with adding and removing "remote" from the name of these functions.

1. This function calls WebPage::releaseAllRemoteImageResources()
2. Which calls RemoteResourceCacheProxy::releaseAllNativeImages()
3. Which calls RemoteRenderingBackendProxy::releaseAllRemoteNativeImages()
4. Which sends the message RemoteRenderingBackend::ReleaseAllNativeImages()

Reply:
I will make them all consistent.